### PR TITLE
optimize match in service storage

### PR DESCRIPTION
**Pull request by firedpeanut**
better read

### CODE REVIEW

File: linera-service/src/storage.rs [hunk 1]
```diff
@@ -90,16 +90,19 @@ impl StorageConfig {
...
             #[cfg(feature = "rocksdb")]
-            Rocksdb { path } if path.is_dir() => {
-                tracing::warn!("Using existing database {:?}", path);
-                let client = RocksdbStoreClient::new(path.clone(), wasm_runtime, cache_size);
-                job.run(client).await
-            }
-            #[cfg(feature = "rocksdb")]
             Rocksdb { path } => {
-                std::fs::create_dir_all(path)?;
+                let create_dir = if path.is_dir() {
```
Comment (ma2bd) [inline linera-service/src/storage.rs:94]: `create_dir` -> `is_new_database`

Comment (ma2bd) [inline linera-service/src/storage.rs:94]: nit: `create_dir` -> `is_new_dir`

Comment (firedpeanut) [reply linera-service/src/storage.rs]: OK

**Review by ma2bd — APPROVED** [review]
Thanks!

**Pull request merged**